Makefile.maker.yaml

```
@@ -33,3 +33,6 @@ verbatim: |
   # This is for manual testing.
   run-api: build/keppel
     set -euo pipefail && source ./.env && $(CURDIR)/build/keppel server api
+
+  copy-fixtures:
+    fd '.actual' | xargs -I{} bash -c 'mv {} $$(echo {} | sed "s/.actual//g")'
```

fd is a non-standard utility and can easily be replaced by a standard one:
```suggestion
    find -name '*.actual' | xargs -I{} bash -c 'mv {} $$(echo {} | sed "s/.actual//g")'
```